sechub-doc/src/test/java/com/daimler/sechub/restdoc/ProjectAdministrationRestControllerRestDocTest.java

```diff
@@ -273,19 +280,86 @@ public void restdoc_show_project_details() throws Exception {
 							parameterWithName(PROJECT_ID.paramName()).description("The id for project to show details for")
 						),
 				responseFields(
-							fieldWithPath(ProjectDetailInformation.PROPERTY_PROJECT_ID).description("The name of the project"),
-							fieldWithPath(ProjectDetailInformation.PROPERTY_USERS).description("A list of all users having access to the project"),
+							fieldWithPath(ProjectDetailInformation.PROPERTY_PROJECT_ID).description("The name of the project."),
+							fieldWithPath(ProjectDetailInformation.PROPERTY_USERS).description("A list of all users having access to the project."),
 							fieldWithPath(ProjectDetailInformation.PROPERTY_OWNER).description("Username of the owner ofthis project. An owner is the person in charge."),
 							fieldWithPath(ProjectDetailInformation.PROPERTY_WHITELIST).description("A list of all whitelisted URIs. Only these ones can be scanned for the project!"),
 							fieldWithPath(ProjectDetailInformation.PROPERTY_METADATA).description("An JSON object containing metadata key-value pairs defined for this project."),
-							fieldWithPath(ProjectDetailInformation.PROPERTY_METADATA + ".key1").description("An arbitrary metadata key")
+							fieldWithPath(ProjectDetailInformation.PROPERTY_METADATA + ".key1").description("An arbitrary metadata key."),
+							fieldWithPath(ProjectDetailInformation.PROPERTY_DESCRIPTION).description("The project description.")
 						)
 					)
 				);
 
 		/* @formatter:on */
     }
 
+    @Test
+    @UseCaseRestDoc(useCase = UseCaseAdministratorChangesProjectDetails.class)
+    public void restdoc_change_project_description() throws Exception {
```

Comment:
The use case is: `UseCaseAdministratorChangesProjectDetails`, but you are trying to document only the `projecty_description`. I would expect the method to be called `restdoc_change_project_details`.

Reply:
done